.gitlab/includes/common_spack_pipeline.yml

```diff
@@ -8,7 +8,7 @@ include:
   - local: '.gitlab/includes/common_pipeline.yml'
 
 variables:
-  SPACK_COMMIT: develop-2024-09-01
+  SPACK_COMMIT: 78117877e002fcb8afec74044b21adffc7655cd3
```

Comment:
To do:
```suggestion
  SPACK_COMMIT: develop-2024-09-15
```
when available next week.

Reply:
I've updated the commit already now, but CI will fail until the new tag is available.